propulate/propulator.py

```diff
@@ -15,8 +16,10 @@
 from ._globals import DUMP_TAG, INDIVIDUAL_TAG
 from .population import Individual
 from .propagators import Propagator, SelectMin
+from .surrogate import Surrogate
 
 log = logging.getLogger(__name__)  # Get logger instance.
+SURROGATE_KEY: Final[str] = "_s"  # key for Surrogate data in Individual
```

Comment:
```suggestion
SURROGATE_KEY: Final[str] = "_s"  # Key for ``Surrogate`` data in ``Individual``
```
I know this is only a very minor thing but I try to keep everything in the codebase as consistent and clean as possible, including the style of comments etc. :D.